src/Simplify.elm

```diff
@@ -6710,11 +6714,11 @@ type alias ConstructWithOneArgProperties =
 
 {-| Properties of a type with with multiple elements. Includes `EmptiableProperties`.
 -}
-type alias CollectionProperties otherProperties =
```

Comment:
There's a much better option:
Taking `Infer.Resources {}` and having a separate `extractInferResources : Infer.Resources a -> Infer.Resources {}`.

You can see how this is done by looking at `asString` which takes `QualifyResources {}` instead of `QualifyResources a`.

Reply:
Nice, I'll try that :+1: